client/service/src/client/client.rs

```diff
@@ -947,7 +947,7 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 		if notify {
 			// sometimes when syncing, tons of blocks can be finalized at once.
 			// we'll send notifications spuriously in that case.
-			const MAX_TO_NOTIFY: usize = 256;
+			const MAX_TO_NOTIFY: usize = 128;
```

Comment:
That's still needed I believe because we didn't report this to Paritytech. It halves memory usage to 16 GB